### ISSUE TITLE
Folder synchronization 150GB - Unable to abort the sync operation

### ISSUE THREAD

**Issue by Shagequi** [nextcloud/android#5179]
Hi,

When you want to synchronize a folder (of 150GB+) manually (by click on contextual menu of the folder, then "Synchronize"), and start the sync, you're unable to stop/abort the sync. You have to wait until it's finished.

A simple first solution would be to add an UI element allowing to stop the operation at any time. An other solution would be to allow the abort in notifications

What do you think about it ?
Thanks

**Comment by jancborchardt** [comment]
Yep, same as you can "Cancel synchronization" in the action menu again, the notification should have the same action. :)
